tests/logictest/suites/base/06_show/06_0003_show_settings

```diff
@@ -1,10 +1,3 @@
-statement query B
```

Comment:
Can we guarantee the value of `max_threads` or `max_memory_usage` didn't be changed by previous files?

Reply:
How about execute `unset max_threads; unset max_memory_usage` before execute this query?

Reply:
I think the test is useless

Reply:
It's usefule. Because the max_threads and max_memory_usage it's determined automatically. So we can test it in [src/query/service/tests/it/storages/testdata/system-tables.txt](https://github.com/datafuselabs/databend/pull/8883/files#diff-86bc27fced7f8932a180b8f562dc48d053d34ce76ef5511ce54132da842c7d1a), so extract it into logictest.

Reply:
settings belong to a session, now we start a new session per file.

If we set global, the setting value will be global, we must remember to unset the settings before end the file.

Reply:
> settings belong to a session, now we start a new session per file.
> 
> If we set global, the setting value will be global, we must remember to unset the settings before end the file.

Agree. 

I think we need to set a rule for test (for now only reviewers and developers should take note).

> 1. All states used in this use case should be assumed to exist and drop them until we start running the actual test content. 
> 2. Clear all states before this test exits.

e.g.

```sql
-- must drop if exists
drop database if exists a;

create database a;
create table a.b(id int);
show tables from a;

-- must clear
drop database a;

` ` `